### ISSUE TITLE
NMOS Test Suite mock registries are HTTP/1.0

### ISSUE THREAD

**Issue by garethsb-sony** [AMWA-TV/nmos-testing#120]
I don't think the NMOS specs discuss differences between HTTP/1.1 and HTTP/1.0, or indicate a recommendation either way?

However, the fact that the test suite's Registries are currently HTTP/1.0 identified a bug in the HTTP implementation of cpprestsdk, which means it doesn't properly handle HTTP/1.0 servers that don't do ``"Connection: Keep-Alive"``. (Fix available at https://github.com/Microsoft/cpprestsdk/pull/1032.)

So, I'm not opposed to the test suite testing HTTP/1.0... but did it ought at least check HTTP/1.1 as well?

**Comment by andrewbonney** [comment]
Fwiw I had a try replacing the basic Flask web server with Werkzeug (which as noted it at least partially uses anyway). Unfortunately this still returned an HTTP/1.0 response.

**Comment by garethsb-sony** [comment]
There appears to be a ``SERVER_PROTOCOL`` config for Werkzeug if I understand my Googling correctly.

**Comment by andrewbonney** [comment]
Yes, I came across that (at least in something testing related), but it seems to default to HTTP/1.1 anyway

**Comment by andrewbonney** [comment]
I'm wondering if it might be necessary to provide instructions for use with a full WSGI compatible web server in order to achieve this (for example https://uwsgi-docs.readthedocs.io/en/latest/WSGIquickstart.html#deploying-flask). I've not seen anything concrete, but running the 'development' web server may only be capable of HTTP/1.0
